2255.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @param {string[]} words
+ * @param {string} s
+ * @return {number}
+ */
+var countPrefixes = function (words, s) {
+  return (result = words.filter((element) => s.startsWith(element)).length);
```

Comment:
```suggestion
  return words.filter((element) => s.startsWith(element)).length;
```
Your code looks tricky to me.
You can just do this and this way looks easier for readers to understand.